internal/distro/distro.go

```diff
@@ -31,6 +42,14 @@ var (
 	xfsMkfsCmd   = "/usr/sbin/mkfs.xfs"
 )
 
+func DiskByIDDir() string       { return diskByIDDir }
+func DiskByLabelDir() string    { return diskByLabelDir }
+func DiskByPartUUIDDir() string { return diskByPartUUIDDir }
+func OEMDevicePath() string     { return oemDevicePath }
+
+func KernelCmdlinePath() string { return kernelCmdlinePath }
+func OEMLookasideDir() string   { return oemLookasideDir }
```

Comment:
What's the reasoning behind having these be functions that return local variables? Why not just have exported `const`s?

Reply:
1. The strings actually have to be `var`s so they can be overridden at link time, so this protects against inadvertent modification.
2. If we exported the variables directly, and later wanted to add an environment variable override for one of them, we'd need to either
    1. lowercase the variable name and add an accessor, which would (silently!) break build scripts that overrode the old name, or
    2. leave the variable name uppercase and risk having a call site that forgot to use the accessor.

  I originally had exported variables and an `init()` that updated them from the env vars, but that's prone to races if an initializer or init function in another package accessed one of the variables.